test/Entropy.FunctionalTests/MvcTests/LocalizationSampleTest.cs

```diff
@@ -1,7 +1,6 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-#if NET46
```

Comment:
This was ifdefed because localization was broken on netcoreapp at some point but never reverted.